prov/efa/src/efa_mr.c

```diff
@@ -60,93 +60,78 @@ static struct fi_ops efa_mr_cache_ops = {
 int efa_mr_cache_entry_reg(struct ofi_mr_cache *cache,
 			   struct ofi_mr_entry *entry)
 {
-	int fi_ibv_access = IBV_ACCESS_LOCAL_WRITE;
-
-	struct efa_mr *md = (struct efa_mr *)entry->data;
-
-	md->domain = container_of(cache->domain, struct efa_domain,
-				  util_domain);
-	md->mr_fid.fid.ops = &efa_mr_cache_ops;
-
-	md->mr_fid.fid.fclass = FI_CLASS_MR;
-	md->mr_fid.fid.context = NULL;
-
-	if (md->domain->ctx->device_caps & EFADV_DEVICE_ATTR_CAPS_RDMA_READ)
-		fi_ibv_access |= IBV_ACCESS_REMOTE_READ;
-
-	md->mr = ibv_reg_mr(md->domain->ibv_pd, entry->info.iov.iov_base,
-			    entry->info.iov.iov_len, fi_ibv_access);
-	if (!md->mr) {
-		EFA_WARN_ERRNO(FI_LOG_MR, "ibv_reg_mr", errno);
-		return -errno;
-	}
-
-	md->mr_fid.mem_desc = md->mr;
-	md->mr_fid.key = md->mr->rkey;
-	return 0;
+	int ret = 0;
+	/* TODO
+	 * Since, access is not passed as a parameter to efa_mr_cache_entry_reg,
```

Comment:
Worth referencing https://github.com/ofiwg/libfabric/issues/5677 in this comment.

Reply:
Done! Thanks